src/styles/start.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
```

Comment:
There is a typo in `stars.scss`
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/05019179-f398-47e2-8b8b-0b3c54d736eb)